Doc/library/asyncio-task.rst

```diff
@@ -414,6 +414,73 @@ reported by :meth:`asyncio.Task.cancelling`.
    Improved handling of simultaneous internal and external cancellations
    and correct preservation of cancellation counts.
 
+Group Cancellation
+------------------
+
+Cancelling an entire task group is not natively supported by the standard
+library but can be achieved by adding a task to the group that raises an
+exception and ignore that exception:
+
+.. code-block:: python
+
+   import contextlib
+   from asyncio import TaskGroup
+
+   class CancelTaskGroup(Exception):
+      """Exception raised to cancel a task group."""
+
+   @contextlib.asynccontextmanager
+   async def CancellableTaskGroup():
+      """A cancellable task group."""
+      try:
+          async with TaskGroup() as group:
+              yield group
+      except* CancelTaskGroup:
+          print('task group was cancelled')
```

Comment:
I think this helper context manager is unnecessary -- it's an elaboration, but to convey the basic idea we don't need it. We just show an example like this:
```py
try:
    with Taskgroup() as tg:
        ...spawn tasks...
except* CancelTaskGroup:
    pass